numba/cuda/functions.cu

```diff
@@ -0,0 +1,211 @@
+#include "cuda_fp16.h"
```

Comment:
I think `package_data` in `setup.py` will need modifying to ensure that this file is present in installations, otherwise we won't be able to use the wrapper outside of running from source.

Reply:
Also, perhaps `cpp_function_wrappers.cu` would be a more descriptive name for this file?

Reply:
Implemented both comments.